yugabytedb_managed/README.md

```diff
@@ -0,0 +1,70 @@
+# YugabyteDB Managed
+
+## Overview
+
+The [YugabyteDB Managed][1] integration with Datadog provides users with the ability to export metrics from their YugabyteDB Managed clusters into Datadog to monitor and alert on.
```

Comment:
The overview should include:

1. What does your product do in 1-2 sentences? 
2. What will customers get from this integration, and why is it valuable to them?
3. What types of data will your integration send in or pull out, and what's the value of that data?

Reply:
updated.